public/controllers/management/components/management/configuration/log-collection/log-collection-logs.js

```diff
@@ -89,23 +90,23 @@ class WzConfigurationLogCollectionLogs extends Component {
         : [];
     return (
       <Fragment>
-        {currentConfig['logcollector-localfile'] &&
-          isString(currentConfig['logcollector-localfile']) && (
+        {currentConfig[LOGCOLLECTOR_LOCALFILE_PROP] &&
+          isString(currentConfig[LOGCOLLECTOR_LOCALFILE_PROP]) && (
             <WzNoConfig
-              error={currentConfig['logcollector-localfile']}
+              error={currentConfig[LOGCOLLECTOR_LOCALFILE_PROP]}
               help={helpLinks}
             />
           )}
-        {currentConfig['logcollector-localfile'] &&
-        !isString(currentConfig['logcollector-localfile']) &&
-        !(currentConfig['logcollector-localfile']['localfile-logs'] || [])
+        {currentConfig[LOGCOLLECTOR_LOCALFILE_PROP] &&
+        !isString(currentConfig[LOGCOLLECTOR_LOCALFILE_PROP]) &&
+        !(currentConfig[LOGCOLLECTOR_LOCALFILE_PROP][LOCALFILE_LOGS_PROP] || [])
           .length ? (
           <WzNoConfig error="not-present" help={helpLinks} />
         ) : null}
-        {currentConfig['logcollector-localfile'] &&
-        !isString(currentConfig['logcollector-localfile']) &&
-        currentConfig['logcollector-localfile']['localfile-logs'] &&
-        currentConfig['logcollector-localfile']['localfile-logs'].length ? (
+        {currentConfig[LOGCOLLECTOR_LOCALFILE_PROP] &&
+        !isString(currentConfig[LOGCOLLECTOR_LOCALFILE_PROP]) &&
+        currentConfig[LOGCOLLECTOR_LOCALFILE_PROP][LOCALFILE_LOGS_PROP] &&
+        currentConfig[LOGCOLLECTOR_LOCALFILE_PROP][LOCALFILE_LOGS_PROP].length ? (
```

Comment:
This kind of conditional can be replaced by the optional chaining operator. ?.
It would simplify the conditional and improve readability.
e.g.
`currentConfig?.[LOGCOLLECTOR_LOCALFILE_PROP]?.[LOCALFILE_LOGS_PROP]?.length`